### PR TITLE
Edit profiles

### PR DESCRIPTION
**Pull request by desolee**

### CODE REVIEW

File: flinder/flinderserver/views.py [hunk 1]
```diff
@@ -58,7 +58,25 @@ def register(request):
...
+@login_required
+def edit_profiles(request):
+    registered = False
+    # This should get from profiles:  request.user
+    user_profile = UserProfile.objects.get(pk=1)
+    print(user_profile)
+    if request.method == 'POST':
+        user_form = UserProfileForm(request.POST)
+        if user_form.is_valid():
+            user = user_form.save()
```
Comment (desolee) [inline flinder/flinderserver/views.py:73]: It has `FOREIGN KEY constraint failed` error, I can't figure out why

File: flinder/flinderserver/views.py [hunk 2]
```diff
@@ -128,7 +146,7 @@ def upload_photos(request):
     return response
 
 
-@login_required
+# @login_required
 def edit_profile(request):
```
Comment (space928) [inline flinder/flinderserver/views.py:150]: If you're replacing the existing edit_profile view then please remove the old one

File: flinder/templates/flinder/edit_profile.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+{% block main_class %} my-auto {% endblock %}
+
+{% block bg_class %}
+    bg_dark
+{% endblock %}
+
+{% block body_block %}
+    <div class="my-5">
+        <div class="col-11 col-sm-5 col-lg-3 col-xxl-2 container">
+            <h1 class="my-5 text-center">Create your account</h1>
```
Comment (space928) [inline flinder/templates/flinder/edit_profile.html:20]: Change the text to read: "Edit your profile"

File: flinder/templates/flinder/edit_profile.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            <form id="login_form" method="post" action="{% url 'flinder:edit_profile' %}">
+                {% csrf_token %}
+                {{ user_form|crispy }}
+                <div class="row justify-content-center">
+                    <input class="btn btn-primary w-auto my-1" type="submit" value="Continue"/>
```
Comment (space928) [inline flinder/templates/flinder/edit_profile.html:25]: Change the value to read: "Save"

**Review by space928 — CHANGES_REQUESTED** [review]
Just a few small things to fix.

File: flinder/templates/flinder/edit_profile.html (hunk 4 above)
Comment (desolee) [reply flinder/templates/flinder/edit_profile.html]: Done

File: flinder/flinderserver/views.py (hunk 2 above)
Comment (desolee) [reply flinder/flinderserver/views.py]: Done

**Review by space928 — APPROVED** [review]

**Pull request merged**